src/files.ts

```diff
@@ -65,7 +63,7 @@ export function parseFileIgnores(path: string): string[] {
 
     rules = f
       .split('\n')
-      .map(l => l.trim().replace(/\/$/, '')) // Remove white spaces and trim slashes
```

Comment:
question: why don't we need this anymore?

Reply:
We don't want to remove the trailing slashes at this point as it messes up with our ability later on to decide the correct parsing from gitignore format to glob.